clusters/app.ci/prow/03_deployment/retester.yaml

```diff
@@ -38,6 +38,7 @@ spec:
         - --job-config-path=/etc/job-config
         - --cache-file=/cache/backoff
         - --enable-on-org=openshift
+        - --enable-on-org=openshift-knative
```

Comment:
@hongkailiu -> our new retester. Are we ready to accept more orgs or should we suggest using the old one in this case?

Reply:
Yes. Let us do it.